src/number_theory/modular_forms/jacobi_theta.lean

```diff
@@ -92,3 +103,82 @@ begin
     congr' 1,
     ring_nf }
 end
+
+lemma jacobi_theta_has_sum_nat (τ : ℍ) :
```

Comment:
can you let the name of the lemma match the order of words in the statement? Like `has_sum_nat_jacobi_theta`. Same thing for many lemmas below (`jacobi_theta_sub_one_is_O_im_infty` and `jacobi_theta_differentiable_at` and `jacobi_theta_mdifferentiable` and `jacobi_theta_continuous`). For instance, we have in the library a lemma `continuous_sin`, not `sin_continuous`.

Reply:
OK, I've renamed them.